web/client/plugins/QueryPanel.jsx

```diff
@@ -179,16 +180,17 @@ const tocSelector = createSelector(
         (state) => state.layers && state.layers.settings || {expanded: false, options: {opacity: 1}},
         (state) => state.controls && state.controls.queryPanel && state.controls.queryPanel.enabled || false,
         state => mapLayoutValuesSelector(state, {height: true}),
+        isDashboardAvailable,
         appliedFilterSelector,
         storedFilterSelector,
         (state) => state && state.query && state.query.isLayerFilter,
         selectedLayerLoadingErrorSelector
-    ], (enabled, groups, settings, querypanelEnabled, layout, appliedFilter, storedFilter, advancedToolbar, loadingError) => ({
+    ], (enabled, groups, settings, querypanelEnabled, layoutHeight, dashboardAvailable, appliedFilter, storedFilter, advancedToolbar, loadingError) => ({
```

Comment:
Why did you changed the name here from layout to layoutHeight?

Reply:
It was misleading; the property contains only layout height.

Reply:
ok